<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quizv2Class</class>
 <widget class="QMainWindow" name="quizv2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>quizv2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="questionList"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>241</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="easyButton">
       <property name="text">
        <string>Easy</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="mediumButton">
       <property name="text">
        <string>Medium</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="hardButtom">
       <property name="text">
        <string>Hard</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="aLineEdit"/>
       </item>
       <item>
        <widget class="QCheckBox" name="acheck">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="bLineEdit"/>
       </item>
       <item>
        <widget class="QCheckBox" name="bcheck">
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="dLineEdit"/>
       </item>
       <item>
        <widget class="QCheckBox" name="dcheck">
         <property name="text">
          <string>D</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="cLineEdit"/>
       </item>
       <item>
        <widget class="QCheckBox" name="ccheck">
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="submitButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finishButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Finish</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>241</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="questionLabel">
       <property name="text">
        <string>Question</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="questionLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="answerALabel">
       <property name="text">
        <string>Answer A</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="answerALineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="answerBLabel">
       <property name="text">
        <string>Answer B</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="answerBLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="answerCLabel">
       <property name="text">
        <string>Answer C</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="answerCLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="answerDLabel">
       <property name="text">
        <string>Answer D</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="answerDLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="correctAnswerLabel">
       <property name="text">
        <string>Correct Answer</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="correctAnswerLineEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="difficultyLabel">
       <property name="text">
        <string>Difficulty</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="difficultyLineEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="pointsLabel">
       <property name="text">
        <string>Points</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="pointsLineEdit"/>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="quizv2.qrc"/>
 </resources>
 <connections/>
</ui>
